<?xml version='1.0'?>
<launch>
	<node pkg="egh450_target_solvepnp" type="pose_estimator" name="pose_estimator" clear_params="true" output="screen">

		<!-- Setting this to true will tell the program to use "~image_raw" + "/compressed" instead -->
		<param name="use_compressed" value="true" type="bool"/>

		<remap from="~camera_info" to="/depthai_node/camera/camera_info"/>
		<remap from="~image_raw" to="/depthai_node/image/raw"/>
		<remap from="~image_raw/compressed" to="/depthai_node/detection/compressed"/>
	</node>
</launch>
